sinatra-contrib/Gemfile

```diff
@@ -16,8 +16,10 @@ group :development, :test do
   end
 
   platform :jruby, :ruby do
-    gem 'slim', '2.1.0'
+    gem 'hamlit'
+    gem 'hamlit-block', '>= 0.7.1'
     gem 'liquid', '~> 2.6.x'
+    gem 'slim'
```

Comment:
slim 2.1.0 is super old and relies on temple ~> 0.6.9. Because hamlit relies on newer temple, I wanted to unpin the slim version. (Since the line is modified, I also moved the line so that they're ordered alphabetically)